### PR TITLE
#176922762 ft(questions): Integrating FAQ

### PR DESCRIPTION
**Pull request by SamuelSabasNayo**
#### What does this PR do?
- Allow users to read FAQs from the database.
- Allow users to read more on one FAQ.
- Allow Super Admin to delete a FAQ in the database.

#### Description of Task to be completed? 
- Integrate frontend to the backend

#### How should this be manually tested?
- Clone the repo 

- In your home directory  cd this branch **`ft-integrate-FAQ-#176922762`**

- Then run **`npm install`** to install all dependencies

- Run **`npm start`** to start the server

- In your default browser navigate to **`http://localhost:3000/FAQ`**

#### Any background context you want to provide?

- To successfully read FAQ, put REACT_APP_BACKEND_URL in `.env` file
- To successfully delete FAQ, **signin as Super Admin**.

#### What are the relevant Pivotal Tracker stories?

- [#176922762](https://www.pivotaltracker.com/story/show/176922762)

#### Screenshots (if appropriate)

Screen Shot 2021-02-13 at 13 42 52
Screen Shot 2021-02-17 at 12 28 22

#### Questions:

- N/A

### CODE REVIEW

**Review by SimonImanigirimpuhwe — COMMENTED** [review]
Good job @SamuelSabasNayo , could you please complete the feature... Since you are integrating `FAQ` consider working on reading more(page of read full text)

**Comment by SamuelSabasNayo** [comment]
> Good job @SamuelSabasNayo , could you please complete the feature... Since you are integrating `FAQ` consider working on reading more(page of ready full text)

Sure, I am working on that, and I will raise other PR concerning more information on a question.

**Comment by SimonImanigirimpuhwe** [comment]
> > Good job @SamuelSabasNayo , could you please complete the feature... Since you are integrating `FAQ` consider working on reading more(page of ready full text)
> 
> Sure, I am working on that, and I will raise other PR concerning more information on a question.

So you decided to make two different PRs?

**Comment by SamuelSabasNayo** [comment]
> > > Good job @SamuelSabasNayo , could you please complete the feature... Since you are integrating `FAQ` consider working on reading more(page of ready full text)
> > 
> > 
> > Sure, I am working on that, and I will raise other PR concerning more information on a question.
> 
> So you decided to make two different PRs?

Yes! @SimonImanigirimpuhwe

**Review by SimonImanigirimpuhwe — APPROVED** [review]

**Review by ConstantineMaliza — COMMENTED** [review]
Great work @SamuelSabasNayo  complete the whole feature let me know then I can approve your work

**Review by leny62 — CHANGES_REQUESTED** [review]
Kindly, check what's causing tests to fail.

**Comment by SamuelSabasNayo** [comment]
> Kindly, check what's causing tests to fail.

Resolved!

**Comment by SamuelSabasNayo** [comment]
> Great work @SamuelSabasNayo complete the whole feature let me know then I can approve your work

Work is ready.

**Review by leny62 — APPROVED** [review]

File: src/views/FAQ/deleteFAQ.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+                  {questions.map((question, index) => (
+                    <div key={question.id} className={classes.content} style={{ display: (index + 1)%2 !== 0 ? 'block' : 'none' }}>
+                      <Badge badgeContent={index + 1} color="primary" classes={{badge: classes.badge}}/>
+                      <Typography variant="h6" className={classes.title}>{question.subject}</Typography>
+                      <Typography className={classes.body}>
+                        {question.message.slice(0, 250) + '...'}
+                      </Typography>
+                      <div style={{ display: 'flex', justifyContent: 'space-between', width: '96%' }}>
+                        <Typography className={classes.body} style={{ margin: '20px 0 0 10px' }}>
+                          Asked by <span className={classes.infos}  style={{ color: '#127CDD' }}>{question.name}</span> on <span className={classes.infos}>{question.createdAt}</span>
```
Comment (ConstantineMaliza) [inline src/views/FAQ/deleteFAQ.js]: Can change this date `toLocaleDateString()` to readable well instead of those time zone

Comment (SamuelSabasNayo) [reply src/views/FAQ/deleteFAQ.js]: Resolved!

**Review by ConstantineMaliza — APPROVED** [review]

**Review by furebo — APPROVED** [review]

**Review by kabano89 — APPROVED** [review]

**Pull request merged**